main.js

```diff
@@ -213,8 +243,8 @@ const packagesForm = () => {
 
 const startApp = () => {
   pinnedRepoCardsOnDom(pinnedRepoArray);
+  projectsOnDom(projectsArray);
```

Comment:
We're gonna have this in it's own "startApp" function that will be triggered by the filter! I set it up in my push request.